src/App.tsx

```diff
@@ -1,173 +1,24 @@
-import { Loader } from './components/Loader';
-
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { HomePage } from './components/HomePage';
+import { Navbar } from './components/Navbar';
+import { PeoplePage } from './components/PeoplePage';
+import { PageNotFound } from './components/PageNotFound/PageNotFound';
 import './App.scss';
 
 export const App = () => (
   <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
+    <Navbar />
 
     <main className="section">
-      <div className="container">
```

Comment:
don't delete this container
![image](https://github.com/mate-academy/react_people-table-basics/assets/91411692/15509db6-2e4c-4db6-a358-6004630946c3)